pkg/apis/lbaas/v2/load_balancer_types.go

```diff
@@ -0,0 +1,105 @@
+package v2
+
+import (
+	"encoding/json"
+
+	"go.anx.io/go-anxcloud/pkg/apis/common"
+	"go.anx.io/go-anxcloud/pkg/apis/common/gs"
+	"go.anx.io/go-anxcloud/pkg/utils/pointer"
+)
+
+// anxcloud:object
+
+// LoadBalancer holds the information of a load balancing configuration within a Cluster
+type LoadBalancer struct {
+	gs.GenericService
+	gs.HasState
+
+	Identifier      string                  `json:"identifier,omitempty" anxcloud:"identifier"`
+	Name            string                  `json:"name,omitempty"`
+	Generation      int                     `json:"generation,omitempty"`
+	Cluster         *common.PartialResource `json:"cluster,omitempty" anxcloud:"filterable"`
+	FrontendIPs     *gs.PartialResourceList `json:"frontend_ips,omitempty"`
+	SSLCertificates *gs.PartialResourceList `json:"ssl_certificates,omitempty"`
+	RawDefinition   *string                 `json:"definition,omitempty"`
```

Comment:
let's have a call about this please, I think we should have the `Definition` here and do that encoding into string internally, but I also seem to remember we talked about this before, but I forgot the outcome ^^'

Reply:
Changed to use custom `Marshaler`/`Unmarshaler`